js/graphData.js

```diff
@@ -1,181 +1,149 @@
       tick: {
         format: '%Y-%m-%d %H:%M:%S',
-        rotate: 45
+        rotate: 45,
       },
       label: {
         text: 'Date',
-        position: 'center'
-      }
+        position: 'center',
+      },
     },
 
     // left-hand y axis
     y: {
       label: {
         text: 'Temperature (°C), Percentage (%)',
-        position: 'outer-middle'
-      }
+        position: 'outer-middle',
+      },
     },
 
     // right-hand y axis
     y2: {
       show: true, // add second y-axis
       label: {
-        text: "Watered Today (0 = No, 1 = Yes)",
-        position: 'outer-middle'
+        text: 'Watered Today (0 = No, 1 = Yes)',
+        position: 'outer-middle',
       },
       tick: {
         count: 2, // We don't need float tick values between a "yes" and "no" option
-        values: [0,1] // makes sure values aren't floating points
+        values: [0, 1], // makes sure values aren't floating points
       },
-    }
-  }
+    },
+  },
 });
 
-
-
 // Get data from database, parse into arrays
 async function getJSONData() {
-    var unix_time = [];
-    var date = [];
-    var soil_temp = [];
-    var soil_moisture = [];
-    var air_temp = [];
-    var air_humidity = [];
-    var heat_index = [];
-    var sunlight = [];
-    var watered_today = [];
-    
-    
-    date.push("x");
-    soil_temp.push("Soil Temp (°C)");
-    soil_moisture.push("Soil Moisture (%)");
-    air_temp.push("Air Temp (°C)");
-    air_humidity.push("Humidity (%)");
-    heat_index.push("Heat Index (°C)");
-    sunlight.push("Sunlight (%)");
-    watered_today.push("Watered Today (0/1)");
-    
-    
-    try {
-        $.get("../databaseconnect.php",
-        function (data) {
-            // alert("success");
-            
-            // for (var i in data) {
-            //     unix_time.push(data[i].unix_time);
-            //     date.push(data[i].date);
-            //     soil_temp.push(data[i].soil_temp);
-            //     soil_moisture.push(data[i].soil_moisture);
-            //     air_temp.push(data[i].air_temp);
-            //     air_humidity.push(data[i].air_humidity);
-            //     heat_index.push(data[i].heat_index);
-            //     sunlight.push(data[i].sunlight);
-            //     watered_today.push(data[i].watered_today);
-            // }
-    
-        })
-        .done(function(data) {
-            
-            for (var i in data) {
-                unix_time.push(data[i].unix_time);
-                date.push(data[i].date);
-                soil_temp.push(data[i].soil_temp);
-                soil_moisture.push(data[i].soil_moisture);
-                air_temp.push(data[i].air_temp);
-                air_humidity.push(data[i].air_humidity);
-                heat_index.push(data[i].heat_index);
-                sunlight.push(data[i].sunlight);
-                watered_today.push(data[i].watered_today);
-            }
-            // alert("Second success");
-        })
-        .fail(function() {
```

Comment:
The error thrown for the fail block here will be handled by the catch block in the updated code